### PR TITLE
add validator livenessprobe and update odin validator image

### PR DESCRIPTION
**Pull request by area363**
SSIA

### CODE REVIEW

File: charts/all-in-one/templates/validator.yaml [hunk 1]
```diff
@@ -167,17 +163,29 @@ spec:
...
           name: gossip
           protocol: TCP
+        livenessProbe:
+          exec:
+            command:
+            {{- if eq $.Values.global.networkType "Main"  }}
+            - /bin/liveness_probe_validator.sh
+            {{- else }}
+            - /bin/liveness_probe_validator.sh
+            {{- end }}
```
Comment (sky1045) [inline charts/all-in-one/templates/validator.yaml:173]: it looks same for both condition

Comment (eseiker) [inline charts/all-in-one/templates/validator.yaml]: is this branch needed?

File: 9c-main/multiplanetary/network/9c-network.yaml [hunk 2]
```diff
@@ -88,6 +88,9 @@ seed:
 validator:
   count: 4
 
+  # custom image for odin validator (remove upon next release)
+  tag: "git-e166707f4ebe7d3ad64318aca458054887a288ba"
```
Comment (eseiker) [inline 9c-main/multiplanetary/network/9c-network.yaml:92]: tag -> image.tag?

File: charts/all-in-one/templates/validator.yaml (hunk 1 above)
Comment (area363) [reply charts/all-in-one/templates/validator.yaml]: Oops, thanks for checking!

File: charts/all-in-one/templates/configmap-probe.yaml [hunk 3]
```diff
@@ -7,5 +7,5 @@ metadata:
     app.kubernetes.io/instance: {{ $.Release.Name }}
 data:
   liveness_probe.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe.sh") . | nindent 4 }}
-  liveness_probe_miner.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe_miner.sh") . | nindent 4 }}
+  liveness_probe_miner.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe_validator.sh") . | nindent 4 }}
```
Comment (eseiker) [inline charts/all-in-one/templates/configmap-probe.yaml:10]: is keeping the file name key being left same intended?

**Review by eseiker — APPROVED** [review]

File: charts/all-in-one/templates/configmap-probe.yaml (hunk 3 above)
Comment (area363) [reply charts/all-in-one/templates/configmap-probe.yaml]: Thanks for the feedback!

**Pull request merged**